SkietbaanBE/SkietbaanBE/Controllers/UserController.cs

```diff
@@ -10,50 +14,64 @@ namespace SkietbaanBE.Controllers
     public class UserController : Controller
     {
         private ModelsContext _context;
-        public UserController(ModelsContext db) {
+        public UserController(ModelsContext db)
+        {
             _context = db;
         }
-
-        //public UserController() { }
-
         // GET: api/User
         [HttpGet]
-        public IEnumerable<User> Get() {
+        public IEnumerable<User> Get()
+        {
             var users = _context.Users.ToArray<User>();
             return users;
         }
-
         // GET: api/User/5
         [HttpGet("{id}", Name = "Get")]
-        public User Get(int id) {
-            User user = _context.Users.Find(id);
+        public async Task<User> Get(int id)
+        {
+            User user = await _context.Users.FindAsync(id);
```

Comment:
Is it not possible to just straight up return the user here instead of first assigning it to a variable?

`return await _context.Users.FindAsync(id);`

Reply:
I jst looked at the code and it seems what i did was unnecessary.  couldve  just return the results straight from the search function

Reply:
@BWMuller  ive created a new pull request where i updated the code with ur suggested changes

Reply:
@BWMuller  please let me know wen u happy with the changes